docs/quick_start_nonhassio_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Quick Start to Home Assistant Data Science for non-hass.io users"
+sidebar_label: "Introduction"
+---
+
+In this quick start guide, we're going to show you how to set up and use [JupyterLab](https://jupyterlab.readthedocs.io/en/stable/), a data science environment. JupyterLab is the tool of choice for data scientists around the globe. Using JupyterLab we will run some reports on your own data. All reports are editable so you can quickly start experimenting and exploring more!
+
+This guide explains the installation and setup of JupyterLab and expects a fresh Ubuntu 18.10 installation. But it should be easy to adapt it for most other platforms.
+
+## Preparing the system
+
+Updating the system and installing required dependencies.
+
+```bash
+sudo apt update
```

Comment:
I don't think that we should go this deep. We don't know if they are on Debian-based Linux, maybe they are using Docker. And I don't want to end up with 1000 variants of installation instructions like we have for Home Assistant, because they keep getting out of date all the time. 

Instead, we should limit it to:

- Install JupyterLab
- Install HASS Data Detective
- Clone https://github.com/home-assistant/home-assistant-notebooks

Reply:
https://www.home-assistant.io/docs/ecosystem/notebooks/installation/ was too short but we can assume that the users who are going down this path know how to keep the system up-to-date.